ApiFirst/ApiFirst/RepositoryList.cs

```diff
@@ -0,0 +1,109 @@
+﻿using DomainModels.Models;
+using DomainServices.Interfaces;
+
+namespace DomainServices.Repositories
+{
+    public class RepositoryList : ICustomerRepository
```

Comment:
```suggestion
    public class CustomerService : ICustomerService
```
Renomear para `CustomerService` e sua interface, `ICustomerService`